docs/applications/pyserver/installation.rst

```diff
@@ -1,60 +1,6 @@
 Installation
 ---------------
 
-It is required that you have python's `pip` installed.
-You probably already have this dependency, but if not,
-please visit `the official pip documentation <https://pip.pypa.io/en/stable/installing/>`_ 
-and install pip before you proceed.
-
-Using ``virtualenv``
-~~~~~~~~~~~~~~~~~~~~
-
-It is recommended to use a virtual environment 
-to separate your global python application from 
-the environment (all the dependency-packages) µRaiden
-likes to run in.
-This will fetch the latest release of µRaiden
-directly from the Python package index and install it:
-
-.. code:: bash
-
-    virtualenv -p python3 env
-    . env/bin/activate
-    pip install microraiden
-
-*editable* development installation 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-When you want to develop on the µRaiden codebase,
-it is best to install it in pip's *editable* mode.
-This way, you can edit the source code directly and 
-never worry about re-installing µRaiden - the linked 
-application always reflects the changes you made.
-
-To install µRaiden for development, simply run:
-
-.. code:: bash
-
-    virtualenv -p python3 env
-    . env/bin/activate
-    git clone git@github.com:raiden-network/microraiden.git
-    cd microraiden
-
-    make pip-install-dev
-
-If you don't like our make script install, you can also run
-the commands manually like so:
-
-.. code:: bash
-    pip install -r requirements-dev.txt
-    pip install -e .
-
-Using a global ``pip3`` installation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-This method is not recommended, as it 
-can interfere with your global python installation.
-If you absolutely must, it still works though:
-
-.. code:: bash
-
-    sudo pip3 install microraiden
+Please go through the developer setup tutorial to set up all :ref:`requirements <dev-setup>`.
```

Comment:
You don't have to do this. The instructions above are enough theoretically.

Maybe we should merge this all with the dev-setup, and not call it dev setup anymore.
We should clearly separate developing on/for µRaiden (dev-setup) from using µraiden as a library (pip pypi mirror setup)
It should become clear though, that this is somewhat decoupled from the Javascript library.

Reply:
Well you're right max. Playing everything in the Dev Setup is most likely the best way to solve it.
We could rename it to "installtion" and explain what "pip install microraiden" does. (I'm not really sure what you can do with this installation)

Reply:
I would say that's actually the main installation process people will (should) use, once we have a stable release and declare the API fixed.

You then can use µRaiden as a library and inherit your custom classes (e.g. handling some specific resource) from the `Expensive` [class](https://github.com/raiden-network/microraiden/blob/master/microraiden/proxy/resources/expensive.py#L23) etc.